css/style.css

```diff
@@ -0,0 +1,71 @@
+#header-img {
+  width: 300px;
+}
+#form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+}
+
+ul {
+  display: flex;
+  list-style-type: none;
+  width: 300px;
+  justify-content: space-around;
+}
+a {
+  text-decoration: none;
+}
```

Comment:
Please add a line between selectors

Reply:
- [ ] same here

Reply:
Please add a line between selectors because it is best practice for better readability and to comply with linterns